### ISSUE TITLE
Switch from jQuery to React?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/send#374]
Feel free to share a laugh, then promptly close this out... But it feels like we're reaching the limits of what jQuery can do on the homepage:

<img width="899" alt="firefox_send" src="https://user-images.githubusercontent.com/557895/28836742-0f539e6e-769f-11e7-9ac7-7b42789ba13c.png">

Currently we have about 4 states for the index/upload page, which we're managing with a bunch of calls like:

```js
$('#page-one').attr('hidden', true);
$('#upload-error').attr('hidden', true);
$('#upload-progress').removeAttr('hidden');
```

Not sure if it'd be easier long term to switch to React so we could use JSX and more modular components to manage state+DOM, versus doing a lot of it manually.

I did see https://medium.com/burke-knows-words/introducing-express-react-starter-b6d299206a3a this morning, and it looks interesting, but not sure how we'll it'd integrate with our upload/download/legal pages since we're not currently a single page app.

Or maybe it's too much effort for too little return.

**Comment by ericawright** [comment]
My concern with react previously was for animations - since we had discussed using a lot of them. so far, we are not, but I think they may still come.

**Comment by pdehaan** [comment]
I feel confident enough in the triage discussion that we can close this.
TLDR: Rewriting in React seems excessive.
